swagger-spec/schema_response_actions/list.yaml

```diff
@@ -0,0 +1,309 @@
+                          upload:
+                            href: >-
+                              https://files.osf.io/v1/resources/ytg45jbs3r/providers/osfstorage/
+                            meta: {}
+                          download:
+                            href: >-
+                              https://files.osf.io/v1/resources/ytg45jbs3r/providers/osfstorage/?zip=
+                            meta: {}
+                      registrations:
+                        links:
+                          related:
+                            href: 'https://api.osf.io/v2/users/sfsr2/registrations/'
+                            meta: {}
+                      institutions:
+                        links:
+                          related:
+                            href: 'https://api.osf.io/v2/users/sfsr2/institutions/'
+                            meta: {}
+                          self:
+                            href: >-
+                              https://api.osf.io/v2/users/sfsr2/relationships/institutions/
+                            meta: {}
+                      preprints:
+                        links:
+                          related:
+                            href: 'https://api.osf.io/v2/users/sfsr2/preprints/'
+                            meta: {}
+                    links:
+                      html: 'https://osf.io/zfsr2/'
+                      profile_image: >-
+                        https://secure.gravatar.com/avatar/8a25f4601ac997f113b3145aa5cf907e?d=identicon
+                      self: 'https://api.osf.io/v2/users/sfsr2/'
+              links:
+                self: 'https://api.osf.io/v2/actions/61b9eaf13cddaa026ef7245d/'
+          meta:
+            total: 2
+            per_page: 10
+            version: '2.20'
+          links:
+            self: 'https://api.osf.io/v2/schema_responses/61b9cd62eb66180215222669/actions/'
+            first: null
+            last: null
+            prev: null
+            next: null
+
+
+post:
+  summary: Create a new Schema Response Action
+  description: >-
+    This creates a new Schema Response Action in order to trigger a state transition for a Schema Response.
+
+    #### Returns
+
+    Returns a JSON object with a `data` key containing the representation of the requested
+    Schema Response Actions, if the request is successful.
+
+    #### Errors
+
+    If the request is unsuccessful, an `errors` key containing
+    information about the failure will be returned. Refer to the [list of error codes](#tag/Errors-and-Error-Codes)
+    to understand why this request may have failed.
+  parameters:
+  - in: path
+    type: string
+    name: schema_response_id
+    required: true
+    description: 'The unique identifier of the Registration Schema example `6176c9d45e01f100091d4f94`.'
+  tags:
+    - Schema Response Actions
+  operationId: schema_response_action_read
+  x-response-schema: Schema Response Actions
+  responses:
+    '200':
```

Comment:
for this POST, the response should be a 201 with the newly created Schema Response Action object, right?